sdk/scheduler/src/main/java/com/mesosphere/sdk/api/types/PlanSummaryInfo.java

```diff
@@ -23,7 +23,7 @@ public static PlanSummaryInfo forPlan(final Plan plan) {
 
     @JsonCreator
     public PlanSummaryInfo(
-            @JsonProperty("status") final String name,
+            @JsonProperty("name") final String name,
```

Comment:
Should just remove the annotations. They aren't going to be used. Their presence implies the Scheduler is parsing SummaryInfos received from elsewhere.